src/services/pscVerificationService.ts

```diff
@@ -21,6 +22,17 @@ export const createPscVerification = async (request: Request, transaction: Trans
     if (!sdkResponse) {
         throw createAndLogError(`${createPscVerification.name} - PSC Verification POST request returned no response for transaction ${transaction.id}`);
     }
+
```

Comment:
the test coverage on the PR is below the threshold.  It looks like these lines are the ones that need covering..

Reply:
Coverage increased after adding some tests.